volumes/miovision/sql/table/create-table-volumes_15min_mvt.sql

```diff
@@ -9,40 +9,43 @@
     volume numeric,
     processed boolean,
     CONSTRAINT volumes_15min_mvt_int_uid_dt_bin_class_leg_mvmt_uid_pkey
-    PRIMARY KEY (intersection_uid, datetime_bin, classification_uid, leg, movement_uid)
+    PRIMARY KEY (
+        volume_15min_mvt_uid, intersection_uid, datetime_bin, classification_uid, leg, movement_uid
+    )
 )
 PARTITION BY RANGE (datetime_bin)
 WITH (
-    OIDS=FALSE
+    OIDS = False
 );
 
 ALTER TABLE miovision_api.volumes_15min_mvt OWNER TO miovision_admins;
-GRANT SELECT, REFERENCES, TRIGGER ON TABLE miovision_api.volumes_15min_mvt TO bdit_humans WITH GRANT OPTION;
+GRANT SELECT, REFERENCES, TRIGGER ON TABLE miovision_api.volumes_15min_mvt
+TO bdit_humans WITH GRANT OPTION;
```

Comment:
we can remove WITH GRANT OPTION for bdit_huamns

Reply:
`bdit_humans WITH GRANT OPTION` appears 27 times in 22 files in volumes/miovision/sql ⚠️

Reply:
We could create a new issue to remove those if you want.